retrieval/target.go

```diff
@@ -30,13 +30,16 @@ import (
 
 	clientmodel "github.com/prometheus/client_golang/model"
 
+	"github.com/prometheus/prometheus/config"
 	"github.com/prometheus/prometheus/storage"
 	"github.com/prometheus/prometheus/utility"
 )
 
 const (
-	// InstanceLabel is the label value used for the instance label.
+	// InstanceLabel is the label name used for the instance label.
 	InstanceLabel clientmodel.LabelName = "instance"
+	// HostLabel is internal label name used for host information.
+	HostLabel clientmodel.LabelName = "__host__"
```

Comment:
So the way to go would be to add it to the the vendored `client_golang` of this branch until both changes are in the respective master branches?